src/main/java/duke/Ui.java

```diff
@@ -27,37 +28,37 @@ public static void outro() {
 
     public static void addTask(Task task, int taskListSize) {
         System.out.print(HORIZONTAL_LINE);
-        System.out.println("Got it. I've added this task:\n" +
-                "  " +
-                task +
-                "\nNow you have " +
-                taskListSize +
-                " tasks in the list.");
+        System.out.println("Got it. I've added this task:\n"
+                + "  "
```

Comment:
Good job in ensuring that coding standards are followed when wrapping around statements :)